agora_analytica/data/yle_2019.py

```diff
@@ -178,14 +170,54 @@ def process_data(df: pd.DataFrame) -> Yle2019E:
     """
     Run processing functions for data.
     """
-
+    print("Processing data... ")
```

Comment:
Nöy. Jos debuggauksen vuoksi halutaan kertoa tämä:
```suggestion
    logger.debug("Processing data... ")
```